pkg/network/ebpf/c/protocols/http2/tls-decoding.h

```diff
@@ -0,0 +1,278 @@
+#ifndef TLS_DECODING_H_
+#define TLS_DECODING_H_
+
+#include "bpf_builtins.h"
+#include "helpers.h"
+
+#include "protocols/http/buffer.h"
+#include "protocols/http2/decoding-common.h"
+#include "protocols/http2/maps-defs.h"
+#include "protocols/tls/https-maps.h"
+
+READ_INTO_USER_BUFFER(http2_preface, HTTP2_MARKER_SIZE)
+READ_INTO_USER_BUFFER(http2_frame_header, HTTP2_FRAME_HEADER_SIZE)
+READ_INTO_USER_BUFFER(http2_char, sizeof(__u8))
+READ_INTO_USER_BUFFER(path, HTTP2_MAX_PATH_LEN)
```

Comment:
If you have a pattern, let's stick with it

```suggestion
READ_INTO_USER_BUFFER(http2_path, HTTP2_MAX_PATH_LEN)
```